client/blocks/all-sites/index.jsx

```diff
@@ -12,13 +13,27 @@ import './style.scss';
 
 const noop = () => {};
 
+const IconContainer = styled.div( {
+	alignItems: 'center',
+	alignSelf: 'center',
+	borderRadius: 0,
+	display: 'inline-flex',
+	marginRight: '8px',
```

Comment:
Does it RTL automatically? IIRC we've had some problems with CSS-in-JS and Intl

Reply:
It doesn't:

https://user-images.githubusercontent.com/26530524/196257381-74d7dc1a-27b3-4ae0-b123-e7b20216670f.mov

Reply:
@zaguiini Nice catch! Maybe we can automatically catch this with some magic linting in the future?

Reply:
I'm looking for some emotion plugin that we can hook into Webpack config but no luck so far.

Reply:
@zaguiini Cool, no rush. I'll fix up this instance.